### ISSUE TITLE
Some errors after upgrading Hugo from v0.110 to v0.112

### ISSUE THREAD

**Issue by happytodev** [jpanther/congo#559]
### What happened?

I upgraded these :

go 1.19 -> 1.20.4
hugo 0.110.0 -> 0.112.0

And after that, when I launch `hugo serve`, I had these errors below

### Theme version

v2.5.4

```bash
hugo mod get -u
go: downloading github.com/jpanther/congo/v2 v2.5.4
go: downloading github.com/jpanther/congo v1.6.4
go: upgraded github.com/jpanther/congo/v2 v2.5.2 => v2.5.4
```

### Hugo version

v0.112

### What browsers are you seeing the problem on?

Chrome

### Relevant Hugo log output

```shell
Start building sites …
hugo v0.112.0+extended darwin/arm64 BuildDate=unknown
ERROR 2023/05/23 20:19:58 render of "home" failed: "/Users/happytodev/sites/partenaires/layouts/_default/baseof.html:1:4": execute of template failed: template: index.html:1:4: executing "index.html" at <partial "partials/functions/warnings.html" .Site>: error calling partial: "/Users/happytodev/sites/partenaires/layouts/partials/functions/warnings.html:1:14": execute of template failed: template: partials/functions/warnings.html:1:14: executing "partials/functions/warnings.html" at <.Site.Params.showAppearanceSwitcher>: can't evaluate field Site in type *page.siteWrapper
ERROR 2023/05/23 20:19:58 render of "taxonomy" failed: "/Users/happytodev/sites/partenaires/layouts/_default/baseof.html:1:4": execute of template failed: template: _default/taxonomy.html:1:4: executing "_default/taxonomy.html" at <partial "partials/functions/warnings.html" .Site>: error calling partial: "/Users/happytodev/sites/partenaires/layouts/partials/functions/warnings.html:1:14": execute of template failed: template: partials/functions/warnings.html:1:14: executing "partials/functions/warnings.html" at <.Site.Params.showAppearanceSwitcher>: can't evaluate field Site in type *page.siteWrapper
Built in 33 ms
Error: error building site: render: failed to render pages: render of "taxonomy" failed: "/Users/happytodev/sites/partenaires/layouts/_default/baseof.html:1:4": execute of template failed: template: _default/taxonomy.html:1:4: executing "_default/taxonomy.html" at <partial "partials/functions/warnings.html" .Site>: error calling partial: "/Users/happytodev/sites/partenaires/layouts/partials/functions/warnings.html:1:14": execute of template failed: template: partials/functions/warnings.html:1:14: executing "partials/functions/warnings.html" at <.Site.Params.showAppearanceSwitcher>: can't evaluate field Site in type *page.siteWrapper
➜  partenaires git:(main) ✗
```

**Comment by happytodev** [comment]
in `layouts\_default\baseof.html` file, I have this line 

`{{- partial "partials/functions/warnings.html" .Site -}}`

I removed `Site` and now it works.

{{- partial "partials/functions/warnings.html" . -}}


I don't know why there is 'Site' here.

Perhaps a typo of mine, so I close this issue.

**Comment by jpanther** [comment]
This is fixed in #561
